src/utils/common.ts

```diff
@@ -54,7 +54,7 @@ export function previousNormalAttributeToken(tokens: ReadonlyArray<Token>, index
 export function previousTypeAttributeToken(tokens: ReadonlyArray<Token>, index: number): AttributeToken | undefined {
 	for (let i: number = index - 1; i > 0; i--) {
 		const token: Token | undefined = tokens[i];
-		if (!token || token.type === 'start-attributes') {
+		if (!token || token.type === 'start-attributes' || token.type === 'tag') {
```

Comment:
@OiYouYeahYou Just want to let you know that I needed to add a check for a tag token here, cause this could potentially a case:

```pug
div(type="application/json")
script.
  const notJson = true;
```

Also I see pros and cons for the reverse iteration of the tokens :thinking: 
So currently the last type attribute is taken, but I don't know if this is wanted behavior :thinking: 
What do you think?